### PR TITLE
test(e2e): add backend app type e2e test

### PR DESCRIPTION
**Pull request by efekarakus**
_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: e2e/multi-app-project/multi_app_project_test.go [hunk 1]
```diff
@@ -146,32 +146,19 @@ var _ = Describe("Multiple App Project", func() {
 					AppName:     appName,
 				})
 				Expect(appShowErr).NotTo(HaveOccurred())
-				Expect(len(app.Routes)).To(Equal(1))
 
-				// Call each environment's endpoint and ensure it returns a 200
-				route := app.Routes[0]
-				Expect(route.Environment).To(Equal("test"))
-				// Since the front end was added first, it should have no suffix.
+				Expect(len(app.ServiceDiscoveries)).To(Equal(1))
```
Comment (kohidave) [inline e2e/multi-app-project/multi_app_project_test.go:150]: Can you add a *third* app so we can still test multiple services sharing one ALB?

File: e2e/multi-app-project/multi_app_project_test.go [hunk 2]
```diff
@@ -146,32 +146,19 @@ var _ = Describe("Multiple App Project", func() {
...
+				Expect(app.ServiceDiscoveries[0].Environment).To(Equal([]string{"test"}))
+				Expect(app.ServiceDiscoveries[0].Namespace).To(Equal(fmt.Sprintf("%s.%s.local:80", appName, projectName)))
+
 				if appName == "front-end" {
+					// Load balanced web apps have routes.
+					Expect(len(app.Routes)).To(Equal(1))
+					route := app.Routes[0]
+					Expect(route.Environment).To(Equal("test"))
```
Comment (iamhopaul123) [inline e2e/multi-app-project/multi_app_project_test.go:158]: Should we try to ping the url here? http.Get(route.URL)

File: e2e/multi-app-project/multi_app_project_test.go (hunk 1 above)
Comment (efekarakus) [reply e2e/multi-app-project/multi_app_project_test.go:150]: Sure!

File: e2e/multi-app-project/multi_app_project_test.go (hunk 2 above)
Comment (efekarakus) [reply e2e/multi-app-project/multi_app_project_test.go:158]: I'll consolidate both yours and David's comment above and add a third application where the two lb web apps will have their URLs tested.

File: e2e/multi-app-project/multi_app_project_test.go (hunk 1 above)
Comment (efekarakus) [reply e2e/multi-app-project/multi_app_project_test.go]: Done! added a "www" app

File: e2e/multi-app-project/multi_app_project_test.go (hunk 2 above)
Comment (efekarakus) [reply e2e/multi-app-project/multi_app_project_test.go]: Done!

**Pull request merged**